independent-projects/bootstrap/maven-plugin/src/main/java/io/quarkus/maven/ExtensionDescriptorMojo.java

```diff
@@ -78,5 +95,37 @@ public void execute() throws MojoExecutionException {
         } catch(IOException e) {
             throw new MojoExecutionException("Failed to persist extension descriptor " + output.resolve(BootstrapConstants.DESCRIPTOR_FILE_NAME), e);
         }
+
```

Comment:
@aloubyansky I don't understand this code: it appears to read the file if it exists, and then writes it back. Surely I'm reading this wrong?

Reply:
Not entirely wrong :) At this point, the file that's actually on the disk is missing groupId and artifactId, which makes it more like a template that needs to be complete by the plugin. However, you're right, it could originally be complete, in which case persisting it is useless.

Reply:
But you're reading it from the output folder. Shouldn't it be read from the src folder?

Reply:
I am reading it from the output folder to support filtering of resources, just in case.

Reply:
OK, if it's done on purpose then fine :)